defaults/main/template.yml

```diff
@@ -764,6 +764,7 @@ nginx_config_http_template:
               #   value: Value-X
               #   always: false
             root: /usr/share/nginx/html
+            # alias: /usr/share/nginx/html
```

Comment:
Is this okay like this? Im not sure if i should uncomment this.

Reply:
I'd leave it uncommented. I usually only comment out variables when there's various configuration options.